### ISSUE TITLE
Do you have any plan to release a training code?

### ISSUE THREAD

**Issue by seokyeong94** [Morpheus3000/PIE-Net#3]
Hi, thanks for the great work.

Is there any plan for you to release a training code for the network? It will be a big help for us to continue our research since we'd like to leverage your SOTA network with ours. 

Thanks

**Comment by Morpheus3000** [comment]
Hello,

Thanks for your interest in our work. Due to the company agreement sponsoring the project, I cannot release the training code. However, I make all the necessary parameters and loss function details available through the paper. The network is also available through the inference code. So I would recommend to use those details to recreate the training code. Let me know if you need help with anything else. Good luck!
